File: BackendChallenge/Models/StatusRequest.cs
namespace BackendChallenge.Models
{
    public class StatusRequest
    {
        public string Status { get; set; }
        public int ItensAprovados { get; set; }
        public decimal ValorAprovado { get; set; }
        public string Pedido { get; set; }
    }
}

File: src/dtos/statusRequest.js
class StatusRequest {
  constructor({ status, itensAprovados, valorAprovado, pedido }) {
    this.status = status;
    this.itensAprovados = Number(itensAprovados);
    this.valorAprovado = Number(valorAprovado);
    this.pedido = pedido;
  }
}

module.exports = StatusRequest;
